environments/ithc/ithc_backend_lb_config.yaml

```diff
@@ -37,6 +37,9 @@ gateways:
         component: profile-sync
       - product: rd
```

Comment:
you don't need the environment in the file name, it's already a very log name

Reply:
removed